### ISSUE TITLE
NTR: X-ray lateral neck: epiglottic enlargement (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9018]
New term request X-ray lateral neck: epiglottic enlargement (nadea3_190206152235): Epiglottitis, swollen epiglottis, thumbprint sign

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000289
- label: Epiglottic enlargement
- parents: [Abnormal epiglottis morphology (HP:0005483)](https://hpo.jax.org/app/browse/term/HP:0005483)
- pmid: PMID:21812674
- def: Anlarged and edematous epiglottis. This sign can be observed on a lateral radiograph of the neck and it suggests a diagnosis of acute infectious epiglottitis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9018
- synonyms: Swollen epiglottis(NONE)|Epiglottis thumb sign(NONE)
